jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/LambdaCreator.kt

```diff
@@ -36,15 +38,27 @@ class LambdaCreator internal constructor(
     private val functionCreator: LambdaFunctionCreator
 ) {
     fun createLambda(
+        module: Module,
+        file: PsiFile,
         functionDetails: FunctionUploadDetails,
+        s3Bucket: String
```

Comment:
I thought s3Bucket was part of `FunctionUploadDetails`

Reply:
It was moved out, due to updateSettings doesnt have it.